semconv/schema.go

```diff
@@ -0,0 +1,19 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semconv
+
+// Should we make this stronger type by declaring "type SchemaURLType string" alias
+// and using it everywhere?
```

Comment:
I dunno.  I'm not sure how much value we'd get from such a type.  Are there any useful methods we could hang off of it? Is it likely that users will accidentally give the wrong string value to `WithSchemaUrl()`?

Reply:
I'll skip it for now.
In the future we may add schema URL parsing functions, e.g. splitting the schema url into schema family and schema version number.